webpack.config.js

```diff
@@ -151,6 +158,9 @@ module.exports = {
       //   verbose: true,
       //   exclude: ['webpack-stats.json']
       // }),
+	  new webpack.optimize.CommonsChunkPlugin({
```

Comment:
blanks instead of tabs, because the rest of the code is already written with blanks